src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -182,18 +151,20 @@ public ServerConfigDocument(CommonLoggerI log, Map<String, File> libertyDirPrope
         locationsAndNames = new HashMap<String, String>();
         props = new Properties();
         defaultProps = new Properties();
-
+        if (originalServerXMLFile != null) {
```

Comment:
Why are we safeguarding setting this field to null, when it defaults to null anyway. Seems unnecessary.